Solinea/goldstone-server
# password reset link references container instead of host

To test:

1. Git checkout resetlink
2. Generate HACKING.md.
2. In the "Postfix" section, start with step # 4.
3. Do a "password reset" sequence.  The link in the e-mail message should work.
